manual/tdr-user-administrator.md

```diff
@@ -0,0 +1,71 @@
+# TDR User Administrator Manual
```

Comment:
Could you put this into its own subfolder, so it's slightly separated from the dev documentation?

Then we can add other admin actions to the same folder, and it will be clearer to admins that they don't have to handle the other tasks that are just for developers.